### ISSUE TITLE
Tell about 'docboth' explicitly in README

### ISSUE THREAD

**Issue by chshersh** [phlummox/hup#1]
Thanks for amazing tool! It's really a shame that Hackage often fails to build packages :( For me only your tool `hup` is working for documentation uploading (because I'm using `stack`).

I just want to recommend to tell about `hup docboth` command explicitly in the beginning because I think this is the main command for most users. 

So they need just to type `hup docboth -u USER -p PASSWORD` and everything works!

I tried to run without `-u` option and saw

```
Exception: HttpExceptionRequest Request {
  host                 = "hackage.haskell.org"
  port                 = 443
  secure               = True
  requestHeaders       = [("Content-Type","application/x-tar"),("Content-Encoding","gzip")]
  path                 = "/package/log-warper-1.7.3/docs"
  queryString          = ""
  method               = "PUT"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
}
 IncompleteHeaders
```

which wasn't very helpful.

**Comment by phlummox** [comment]
Nice to hear that someone is using it :)

Thanks for the feedback, I'll take a look at the README and try to make it more explanatory. And perhaps improve the error messages.

**Comment by phlummox** [comment]
Fixed in [0.3.0.0](https://github.com/phlummox/hup/releases/tag/v0.3.0.0). I think the HTTP timeout error was a different issue, but hopefully have improved the error messages for those, also.

**Comment by chshersh** [comment]
Thanks a lot!
